src/Utilities/EmailSender.cs

```diff
@@ -2,24 +2,25 @@
 using System.Net;
 using System.Net.Mail;
 using System.Threading.Tasks;
-using ePiggyWeb.DataBase;
+// ReSharper disable MemberCanBePrivate.Global
+// ReSharper disable UnassignedGetOnlyAutoProperty
```

Comment:
Are these necessary, because if resharper is not used, a bit weird ?

Reply:
It looks really bad without them in resharper though, because all the properties are marked as being bad, because reshaper doesn't catch that these properties are binded through configuration. And since probably we will be showing our project with resharper, I think it's worth keeping them in